.konflux/override-snapshot.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Snapshot
+metadata:
+  name: override-snapshot
```

Comment:
did you consider replacing name with this `generateName: <application-name>-override-snapshot-` ? 

Even if we don't want that, I think the name should have a prefix/suffix with application name